packages/tools/fluid-runner/src/utils.ts

```diff
@@ -0,0 +1,19 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import * as fs from "fs";
+
+/**
+ * Is the given snapshot in JSON format
+ * @param path - path to snapshot file
+ */
+export function isJsonSnapshot(path: string): boolean {
+    const buffer = Buffer.alloc(1);
+    const fd = fs.openSync(path, "r");
+    fs.readSync(fd, buffer, 0, 1, 0);
+    fs.closeSync(fd);
+
+    return buffer.toString() === "{";
```

Comment:
I think you can read whole buffer only once, and then get first byte and test it. Otherwise, we are loading it twice, right?